repositories.bzl

```diff
@@ -139,7 +142,7 @@ cc_proto_library(
     deps = [
         "//external:cc_gogoproto",
```

Comment:
You need to either sync gogoproto or (preferably) replace this and other occurrences with `"@com_github_gogo_protobuf//:gogo_proto_cc",`. Then, it works.

Reply:
done